data/templates/affiliations.jats

```diff
@@ -0,0 +1,35 @@
+$--
+$-- Affiliations
+$--
+$for(affiliation)$
+<aff id="aff-$affiliation.id$">
+$-- wrap affiliation if it has a known institution identifier
+$if(affiliation.group)$
+<institution content-type="group">$affiliation.group$</institution>
+$endif$
+$if(affiliation.department)$
+<institution content-type="dept">$affiliation.department$</institution>
+$endif$
+<institution-wrap>
+$if(affiliation.organization)$
+<institution>$affiliation.organization$</institution>
+$else$
+<institution>$affiliation.name$</institution>
+$endif$
+$if(affiliation.isni)$
```

Comment:
Might it be more future-proof to leave the institution identifiers open and loop over an array of objects with `id` and `id-type` (and perhaps `vocab` and  `vocab-identifier`) properties?

Reply:
This is the *status quo* for all identifiers right now, including orcid, pmid, pmcid, etc. The current variant is consistent with that. We can add the generic variant on top of that.